### PR TITLE
Improve translations

### PR DESCRIPTION
**Pull request by baumanno**
Reword some phrasings which were a bit off, fix typos and unify names.
Includes the compacted translation files.

I have not tested the "importability" of these new strings. Please let me know if I should and I will set up the build chain!

### CODE REVIEW

File: translations/campaigns/campaign_en.csv [hunk 1]
```diff
@@ -1,29 +1,29 @@
 ,en
-MISSION_TITLE_MAYDAY_SQUARE,"MayDay Square"
```
Comment (czlowiekimadlo) [inline translations/campaigns/campaign_en.csv:2]: 'MayDay' is intentional, it should stay that way.

File: translations/campaigns/campaign.csv [hunk 2]
```diff
@@ -1,18 +1,18 @@
 ,en,fr,pl
```
Comment (czlowiekimadlo) [inline translations/campaigns/campaign.csv:1]: there is no need to modify this file in PR

File: translations/translations.csv [hunk 3]
```diff
@@ -182,17 +182,17 @@ MSG_WORKSHOP_LOSE_MAP_DATA,YOU WILL LOSE MAP DATA!,VOUS PERDREZ LA CARTE CRÉÉE
 MSG_WORKSHOP_USE_UNIQUE_NAME,USE UNIQUE NAME,UTILISEZ UN NOM UNIQUE,UŻYJ UNIKALNEJ NAZWY
```
Comment (czlowiekimadlo) [inline translations/translations.csv:182]: there is no need to modify this file in PR

**Review by czlowiekimadlo — CHANGES_REQUESTED** [review]
Thank you for the contribution! Just a couple of comments:
- 'MayDay' is an intentional name, please leave it like that
- only *_en.csv files need to be modified. other ones will be automatically generated by us using the bin/compact.py script - it is easier to review a PR if these are not included

**Comment by czlowiekimadlo** [comment]
When you provide updated _en.csv files, I will import them and check if modified text does not overflow from text areas. Thanks!

**Comment by czlowiekimadlo** [comment]
Also, if you want to test these yourself, all you have to do is to run the compacting script and then run the game from the Godot Editor - it will re-import all translations automatically.

File: translations/campaigns/campaign_en.csv (hunk 1 above)
Comment (baumanno) [reply translations/campaigns/campaign_en.csv:2]: I see, but then the CamelCase-style should be adopted throughout the storyline, I guess? I saw this referenced as "MayDay" and "Mayday", the latter occuring more often, so I went with the majority vote.

**Comment by baumanno** [comment]
Hi there and thanks for the feedback! I revised the original changes and integrated the changes you requested (including camelizing MayDay everywhere). I force-pushed the altered original commit, so if you want to you can simply continue reviewing here :) Cheers!

File: translations/campaigns/campaign_en.csv (hunk 1 above)
Comment (czlowiekimadlo) [reply translations/campaigns/campaign_en.csv:2]: If you encounter any instances of 'Mayday' in translations - these are mistakes and should be fixed.

**Comment by czlowiekimadlo** [comment]
Thank you! I'm in the middle of other work right now, and it's getting a bit late here, so I will test this tomorrow and accept if all of these new texts fit.

**Pull request merged**